frontend/packages/dev-console/src/components/add/AddCardSection.tsx

```diff
@@ -23,22 +25,38 @@ const AddCardsSection: React.FC<AddCardsSectionProps> = ({
   namespace,
   addActionExtensions,
   addActionGroupExtensions,
+  loading,
 }) => {
   const filteredAddActionExtensions: ResolvedExtension<AddAction>[] = filterNamespaceScopedUrl(
     namespace,
     useAccessFilterExtensions(namespace, addActionExtensions),
   );
 
-  const sortedActionGroup: LoadedExtension<AddActionGroup>[] = getSortedExtensionItems<
-    LoadedExtension<AddActionGroup>
-  >(addActionGroupExtensions);
+  const isLoading: boolean = loading || filteredAddActionExtensions.length === 0;
```

Comment:
You check for both values here. loading or filtered extensions is empty.

Maybe it's possible that `useAccessFilterExtensions` returns is loading status as well so that you check for a clean 
access-review loading-status here as well instead of the fact that there is no action available.